tails_files/install.sh

```diff
@@ -19,16 +20,15 @@ apt-get update
 apt-get install -y build-essential
 gcc -o $SCRIPT_BIN securedrop_init.c
 
-# copy launcher and icon
-cp securedrop_icon.png $INSTALL_DIR
-cp SecureDrop\ Init.desktop $PERSISTENT
-
-# copy securedrop_init.py script
+# copy init scripts
 cp securedrop_init.py $SCRIPT_PY
+cp .xsessionrc $INSTALL_DIR
+
+cp securedrop_icon.png $INSTALL_DIR
 
 # prepare torrc_additions
 cp torrc_additions $ADDITIONS
-gedit $ADDITIONS
+gedit $ADDITIONS > /dev/null 2>&1
```

Comment:
It might also be good to add a line (commented with a `#`) to the default torrc_additions that has instructions, e.g.

```
# Enter the HidServAuth lines for your authenticated hidden services here.
# When you're done, save the file and quit the editor.
```

Since sometimes people are confused that they need to quit the editor in order to continue. @ageis What do you think?